.ci/daily/Jenkinsfile

```
@@ -1,6 +1,6 @@
 // https://github.com/camunda/jenkins-global-shared-library
 // https://github.com/camunda/cambpm-jenkins-shared-library
-@Library(['camunda-ci', 'cambpm-jenkins-shared-library']) _
+@Library(['camunda-ci', 'cambpm-jenkins-shared-library@CAM-13267-support-DB2-115']) _
```

```suggestion
@Library(['camunda-ci', 'cambpm-jenkins-shared-library']) _
```
Let's not forget to change it back.